### ISSUE TITLE
An error occurred

### ISSUE THREAD

**Issue by lzqcode** [eli5168/improved_gan_pytorch#1]
this line "output = max_score + torch.log(torch.sum(torch.exp(vec - max_score_broadcast),1))"
RuntimeError: sizes do not match,I am a novice, how to solve this excuse me

**Comment by eli5168** [comment]
Sorry for my late reply.
The definition of log_sum_exp is https://en.wikipedia.org/wiki/LogSumExp
Actually, you can print() these variables step by step.
This way, you can find that the shape of before_softmax_output is 100x10. Similarly, max score is 1, max_score_broadcast is 1x10, and torch.sum(torch.exp(vec - max_score_broadcast),1) is 1.

BTW, you are also suggested to learn those methods in the document. http://pytorch.org/docs/master/torch.html?highlight=torch%20sum#torch.sum
